example/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:erbium-alpine as builder
```

Comment:
Could we use fermium, the currently active node lts

Reply:
Erbium is also the version of node declared in `.nvmrc`. Can we update it there too?

Reply:
Yes